SonarQube.TeamBuild.PostProcessor/Program.cs

```diff
@@ -20,7 +20,9 @@ internal static class Program
 
         private static int Main(string[] args)
         {
-            ConsoleLogger logger = new ConsoleLogger(includeTimestamp: true);
+            ConsoleLogger logger = new ConsoleLogger();
+            Utilities.LogAssemblyVersion(logger, typeof(Program).Assembly, Resources.AssemblyDescription);
+            logger.IncludeTimestamp = true;
```

Comment:
For cosmetic reasons (and consistency with the sonar-runner) the version info is being logged without a timestamp.